packages/website/components/Ecosystem/EcosystemSection.tsx

```diff
@@ -57,6 +57,15 @@ const ecosystemData: EcosystemData[] = [
     filters: [],
     isLive: true,
   },
+  {
+    icon: "/images/ecosystem/chaindrop.png",
+    name: "Chaindrop Faucet",
+    link: "chaindrop.org",
```

Comment:
this path is broken, i think `https://chaindrop.org` will work. we parse this string to make it look pretty, but we need the valid href link.